README.md

```diff
@@ -162,7 +162,7 @@ make tf_init env=<ENVIRONMENT>
 make tf_apply env=<ENVIRONMENT>
 ```
 
-where ENVIRONMENT is one of `dev`, `preprod` or `prod`
+where ENVIRONMENT is one of `dev`, `sandbox`, `preprod` or `prod`
```

Comment:
Do you have 4 envs? We have dev, preprod and prod, where preprod is just another word for sandbox. If this is coming into our repo, we'd rather keep preprod and ditch sandbox